driver/aliases.py

```diff
@@ -87,15 +88,15 @@
           single(sum([g(),weight(hgc,2)])),
           single(sum([g(),weight(hgc,2)]),pref_only=true)],
          boost=1000),
-     preferred=[hcea,hgc],reopen_closed=true,cost_type=one)
-],repeat_last=true,continue_on_fail=true)"""]
+     preferred=[hcea,hgc],reopen_closed=true,cost_type=ONE)
+],repeat_last=true,continue_on_fail=true)))))"""]
 
 def _get_lama(**kwargs):
     return [
         "--if-unit-cost",
         "--evaluator",
```

Comment:
There is still an `--evaluator` here and it caused some problems. The `--search` in the next line is also too late.

Reply:
We should test all aliases ate least once. Currently at least `seq-sat-lama-2011` is not working.

Reply:
already fixed locally in Patrick's code.